interactive_select_printer.go

```diff
@@ -71,11 +72,17 @@ func (p InteractiveSelectPrinter) WithMaxHeight(maxHeight int) *InteractiveSelec
 	return &p
 }
 
+// OnInterrupt sets the function to execute on exit of the input reader
+func (p InteractiveSelectPrinter) OnInterrupt(exitFunc func()) *InteractiveSelectPrinter {
```

Comment:
Sorry, I just noticed this now. Our naming convention for methods, that set an option is, that it has to start with `With`. So in this case, those functions should be called `WithInterruptFunc`, so that the IDE suggest every customizable option, when `.With` is typed.

Reply:
Then lets take a minute to discuss about the naming, as we are introducing more or less a naming convention for callback/hook functions in pterm. Changing this later may cause  a breaking change.

Using `OnInterruptFunc` (as suggested in this PR):

```go
pterm.DefaultInteractiveTextInput.WithOnInterruptFunc(exitfunc)
pterm.InteractiveTextInputPrinter{OnInterruptFunc: func() { os.Exit(1)} )
```

As this is more or less a hook now (as the os.Exit is not used) `hook` can be used too:

```go
pterm.DefaultInteractiveTextInput.WithOnInterruptHook(exitfunc)
pterm.InteractiveTextInputPrinter{OnInterruptHook: func() { os.Exit(1)} )
```

As the quite generic callback:

```go
pterm.DefaultInteractiveTextInput.WithOnInterruptCallback(exitfunc)
pterm.InteractiveTextInputPrinter{OnInterruptCallback: func() { os.Exit(1)} )
```

Or following the current "cancel" terminology used in cancelation signal:

```go
pterm.DefaultInteractiveTextInput.WithCancelFunc(exitfunc)
pterm.InteractiveTextInputPrinter{CancelFunc: func() { os.Exit(1)} )
```

If this does not matter for now, then we can continue with OnInterruptFunc I guess, which is quite descriptive anyway.